api/v1beta1/common_webhook.go

```diff
@@ -115,3 +116,23 @@ func IsUniqMAC(macNodeStatus map[string]OpenStackMACNodeReservation, mac string)
 	}
 	return false
 }
+
+// validateNetworks - validate that for all configured subnets an osnet exists
+func validateNetworks(namespace string, networks []string) error {
+	for _, subnetName := range networks {
+		//
+		// Get OSnet with SubNetNameLabelSelector: subnetName
+		//
+		labelSelector := map[string]string{
+			SubNetNameLabelSelector: subnetName,
+		}
+		osnet, err := GetOpenStackNetWithLabel(namespace, labelSelector)
+		if err != nil && k8s_errors.IsNotFound(err) {
+			return fmt.Errorf(fmt.Sprintf("%s %s instance is supported at this time", osnet.GetObjectKind().GroupVersionKind().Kind, subnetName))
```

Comment:
Should this maybe say `not supported`?

Reply:
I think that message does not make sense here and is likely from c/p. will update it

Reply:
done in new commit